ErgoScript-Developer-Course/batch-0/ahrnsetido/func-prog-hw-02.md

```diff
@@ -0,0 +1,96 @@
+# Homework 2
+
+## Recap basic ErgoScript and programming
+
+### Example 1
+
+This example contains simple Data Type definitions
+- [x] Long
+- [x] Byte
+
+```scala 
+  val ErgToNanoErg: Long = 1000000000L
+  val tokenPriceErg: Long = 20 * ErgToNanoErg
+  val thisYear: Byte = 2022
+  val thisMonth: Byte = 4
+```
+### Example 2
+
+This example checks if the value in the input box INPUTS(0) is larger than tokenPriceErg (20 Erg), and contains:
+- [x] If else
+- [x] Boolean
+
+```scala 
+  val userBox: Box = INPUTS(0)
+
+  if(userBox.value > tokenPriceErg ){
+    val enoughErg: Boolean = true
+  }else{
+    val enoughErg: Boolean = false
```

Comment:
Same issue as my first comment.

A way to fix this would be to set the variable equal to the if/else statement expression. Something like this:
`val enoughErg: Boolean = (userBox.value > tokenPriceErg)`

This does the same thing as before.